lib/transformer.go

```diff
@@ -49,11 +50,17 @@ func Transform(ctx context.Context, plan plantypes.Plan, preExistingPlan bool, o
 	requirements, _ := selectorsInPlan.Requirements()
 	transformerSelectorObj = transformerSelectorObj.Add(requirements...)
 
+	remoteOutputFSPath := vcs.GetInputClonedPath(outputPath, common.RemoteOutputsFolder, true)
```

Comment:
Can you change the function name `GetInputClonedPath` to something relevant to the usecase as I see this is being used widely now. May be `GetClonedPath` or something?

Reply:
Hello @kmehant I have addressed this in https://github.com/konveyor/move2kube/pull/1031